packages/web/stores/user-settings/show-dust.tsx

```diff
@@ -23,10 +23,14 @@ export class ShowDustUserSetting implements IUserSetting<ShowDustState> {
   };
 
   constructor(fiatSymbol: string) {
-    this.displayLabel = `Show pools/assets < ${fiatSymbol} 0.01`;
+    this.fiatSymbol = fiatSymbol;
```

Comment:
Low prio: we can save some lines of code by declaring a readonly property directly in the constructor as `protected readonly fiatSymbol: string`

It removes the need to declare in class body and initialize in constructor.